# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bsd-games/compile-fix.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur bsd-games-2.17.orig/hack/hack.termcap.c bsd-games-2.17/hack/hack.termcap.c
--- bsd-games-2.17.orig/hack/hack.termcap.c	2005-02-15 03:31:35.000000000 +0200
+++ bsd-games-2.17/hack/hack.termcap.c	2005-10-15 16:25:25.000000000 +0300
@@ -80,7 +80,7 @@
 char           *HO, *CL, *CE, *UP, *CM, *ND, *XD, *BC, *SO, *SE, *TI, *TE;
 static char    *VS, *VE;
 static int      SG;
-char            PC = '\0';
+char            PCC = '\0';
 char           *CD;		/* tested in pri.c: docorner() */
 int             CO, LI;		/* used in pri.c and whatis.c */
 
@@ -101,7 +101,7 @@
 	if (tgetent(tptr, term) < 1)
 		error("Unknown terminal type: %s.", term);
 	if ((pc = tgetstr("pc", &tbufptr)) != NULL)
-		PC = *pc;
+		PCC = *pc;
 	if (!(BC = tgetstr("bc", &tbufptr))) {
 		if (!tgetflag("bs"))
 			error("Terminal must backspace.");
